### PR TITLE
Add generic Docker skeleton for detected Python apps (`convox init`)

### PR DESCRIPTION
**Pull request by soulshake**
Test suite passes, but still needs manual verification.

### CODE REVIEW

File: cmd/convox/templates/init/python/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: "2"
+services:
+  web:
+    build: .
+    labels:
+      - convox.port.443.protocol=tls
+      - convox.port.443.proxy=true
```
Comment (mattmanning) [inline cmd/convox/templates/init/python/docker-compose.yml]: This line enables PROXY protocol. Will the gunicorn backend be able to handle that? We might need to run something in front. In our rails image, for example we use nginx to read the PROXY protocol headers and set HTTP headers for the app backend:

https://github.com/convox/rails/blob/master/conf/nginx.conf#L35-L38
https://github.com/convox/rails/blob/master/bin/web#L6-L7

File: cmd/convox/templates/init/python/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    links:
+      - database
+    ports:
+      - "80:5000"
```
Comment (mattmanning) [inline cmd/convox/templates/init/python/docker-compose.yml]: Is it typical for flask backends to listen on ports 5000 and 8000?

**Comment by mattmanning** [comment]
[Add generic Docker skeleton for detected Python apps (`convox init`)](https://trello.com/c/pHMAviur/781-add-generic-docker-skeleton-for-detected-python-apps-convox-init)

File: cmd/convox/templates/init/python/docker-compose.yml (hunk 2 above)
Comment (soulshake) [inline cmd/convox/templates/init/python/docker-compose.yml]: Port 5000 is the default, according to the [flask.Flask.run() doc](http://flask.pocoo.org/docs/0.11/api/#flask.Flask.run).

Port 8000, no.

However, this isn't meant to be Flask-specific, only Python-specific, and I believe 5000/8000 for http/https is common for Python apps.

**Comment by ddollar** [comment]
This needs rebase, etc

**Comment by soulshake** [comment]
[Deal with `convox init` Python PR](https://trello.com/c/MpYbboDo/1437-deal-with-convox-init-python-pr)

**Pull request closed without merging**